package.json

```
@@ -14,21 +11,19 @@
   },
   "license": "MIT",
   "author": "stylelint",
-  "type": "module",
   "scripts": {
-    "build": "npm run gendoc && cd website && NODE_ENV=production npm run build",
-    "build:ci": "npm run gendoc && cd website && npm ci && NODE_ENV=production npm run build",
+    "build": "npm run gendoc && docusaurus build",
+    "docusaurus": "docusaurus",
     "format": "prettier . --write",
     "pregendoc": "rimraf docs",
-    "gendoc": "node ./scripts/generate-stylelint-docs docs",
-    "postinstall": "cd website && npm install",
+    "gendoc": "node ./scripts/generate-docs.mjs docs",
     "lint": "npm-run-all --parallel lint:*",
     "lint:css": "stylelint \"**/*.css\" --ignore-path .gitignore ",
     "lint:formatting": "prettier . --check",
     "lint:js": "eslint . --ignore-path .gitignore",
```

[suggestion] It seems we need to change the default [`--ext`](https://eslint.org/docs/user-guide/command-line-interface#-ext) value of ESLint as below:

```suggestion
    "lint:js": "eslint . --ignore-path .gitignore --ext .js,.mjs",
```